CMakeLists.txt

```diff
@@ -68,37 +66,38 @@ elseif(DOXYGEN_FOUND)
 endif(DOXYGEN_FOUND)
 
 if (BUILD_PYTHON)
+  # This is NOT the modern cmake-included FindPython!
```

Comment:
what do you mean?

Reply:
There's a custom CMake/FindPython in LOFARBeam, while modern CMake includes its own FindPython. To anyone reading CMakeLists.txt it seems this line is calling the CMake standard FindPython module, while in fact it's calling the one included with LOFARBeam.

Reply:
Ooh, good point. Is there a reason we use our own?

Reply:
I think not. Note that on the EveryBeam we've already switched to pybind11 on one of the branches, so these changes to cmake should be deprecated quite soon.